.github/workflows/test.yaml

```diff
@@ -44,3 +44,15 @@ jobs:
 
       - name: Run chart-testing (install)
         run: ct install --config .github/ct.yaml
+
+      - name: Build chart package
+        run: |
+          changed=$(ct list-changed --config .github/ct.yaml)
+          [ $(echo -n "$changed" | grep -c '^') == 1 ] && helm package $changed || echo "skipping package"
+
+      - name: Upload helm chart
+        uses: actions/upload-artifact@v3
+        with:
+          path: ./*.tgz
+          retention-days: 30
+          if-no-files-found: ignore
```

Comment:
what do we test here?
feels like it could be a separate job

Reply:
it is not a test, but a way to get helm package to test later. It just saves built package so anyone could test it